### PR TITLE
Added Readme and ex1.rb

### PR DESCRIPTION
**Pull request by JonathanSpeek**
[#104389358]

### CODE REVIEW

**Comment by jasonnoble** [comment]
Missing Tracker Story # on second commit...

File: chapter_1/ex1.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# puts 'Hello World!'
+# puts 'Hello Again'
+# puts 'I like typing this.'
+# puts 'This is fun.'
+# puts 'Yay! Printing.'
+# puts "I'd much rather you 'not'."
```
Comment (jasonnoble) [inline chapter_1/ex1.rb]: Question for ya... Why don't lines 6 and 7 need to be switched to single quotes?  Why is Rubocop ok with that?

**Comment by jasonnoble** [comment]
Needs rebasing...

File: chapter_1/ex1.rb (hunk 1 above)
Comment (JonathanSpeek) [inline chapter_1/ex1.rb]: Because there is a single quote in the string.  I could also use / to escape it.

**Pull request merged**